cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -38,4 +38,10 @@
   public static final String SCHEDULE_NAME = "scheduleName";
```

Comment:
can this be schedule.name? And with namespaces (do we have that yet?), would this contain the namespace? Or is it always the schedule name, because that is within the scope of the application?